### ISSUE TITLE
Naming of queues in testrunner.py is confusing

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt#13413]
There is a pair of queues used to communicate between `TestRunnerManager` and `TestRunner` in testrunner.py, created here, where `self` is the `TestRunnerManager` instance:
https://github.com/web-platform-tests/wpt/blob/08d0c7d0de6c135b8f11364f6d0175981524f530/tools/wptrunner/wptrunner/testrunner.py#L302-L303

`TestRunner` has slightly different names, and initializes them here:
https://github.com/web-platform-tests/wpt/blob/08d0c7d0de6c135b8f11364f6d0175981524f530/tools/wptrunner/wptrunner/testrunner.py#L58-L59

They came from here, same names in the same order but prefixed with `runner_`:
https://github.com/web-platform-tests/wpt/blob/08d0c7d0de6c135b8f11364f6d0175981524f530/tools/wptrunner/wptrunner/testrunner.py#L120

*However*, this is where the arguments for `start_runner` are set up:
https://github.com/web-platform-tests/wpt/blob/08d0c7d0de6c135b8f11364f6d0175981524f530/tools/wptrunner/wptrunner/testrunner.py#L483-L484

In other words:
- `TestRunnerManager.remote_queue` = `TestRunner.command_queue`
- `TestRunnerManager.command_queue` = `TestRunner.result_queue`

That the two `command_queue`s were not the same was not at all obvious to me, and made it a lot harder to understand https://github.com/web-platform-tests/wpt/issues/13407.

@jgraham

**Comment by foolip** [comment]
Ideas:
 - name the queues `command_queue` and `result_queue` in both classes, and have them match up
 - use a [Pipe](https://docs.python.org/2/library/multiprocessing.html#multiprocessing.Pipe)

**Comment by jgraham** [comment]
FWIW the logic is that the name is always true from the point of view of the object. So basically the `command_queue` is always the one that you `get()` messages from in your event loop. That doesn't mean it's not confusing of course, but it does have an internal logic, and reusing the same name across the two objects would break that.

Using a `Pipe` also seems like a bad idea; a `Queue` is literally an abstraction over a `Pipe` that provides buffering and message-based semantics.

**Comment by foolip** [comment]
> FWIW the logic is that the name is always true from the point of view of the object. So basically the `command_queue` is always the one that you `get()` messages from in your event loop. That doesn't mean it's not confusing of course, but it does have an internal logic, and reusing the same name across the two objects would break that.

I did suspect this *after* I realized my mistake of equating the two `command_queue`s. Is it accurate, though, to say that `TestRunnerManager` is getting commands from `TestRunner`? It should just be sending results and logs, is there any "do this for me" sort of communication in the runner&rarr;manager direction?

Whether named the same across the two classes or not, do you have names you think would be clearer?

> Using a `Pipe` also seems like a bad idea; a `Queue` is literally an abstraction over a `Pipe` that provides buffering and message-based semantics. 

Alright, I thought a queue was a 1-way channel and a pipe a 2-way channel, based on reading only this:
https://docs.python.org/2/library/multiprocessing.html#exchanging-objects-between-processes

**Comment by foolip** [comment]
I'm not gonna fix this :)
